CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+### Features
+- Add support for Sentry Kotlin Compiler Plugin ([#2695](https://github.com/getsentry/sentry-java/pull/2695))
+  - In conjunction with our sentry-kotlin-compiler-plugin we improved Jetpack Compose support for
```

Comment:
nit: Ideally here would be also a link to sentry-kotlin-compiler-plugin docs, but we can update it later ;)

Reply:
I'd really appreciate a link to the compiler plugin docs as I've been meaning to try these new features and haven't been able to understand how to use the compiler plugin.

Reply:
@tfcporciuncula they are in the making, but soon will be there :) https://github.com/getsentry/sentry-docs/pull/7026